src/mumble/MainWindow.cpp

```diff
@@ -524,8 +525,12 @@ void MainWindow::closeEvent(QCloseEvent *e) {
 			showMinimized();
 			e->ignore();
 
-			if (qcbRemember->isChecked()) {
-				Global::get().s.connectedQuitBehavior = ConnectedQuitBehavior::MINIMIZE;
+			/* If checkbox is checked and not connected, keep asking when connected
+			 * If checkbox is checked and connected, always minimize when connected */
+			if (qcbRemember->isChecked() && !(sh && sh->isRunning())) {
+				Global::get().s.quitBehavior = QuitBehavior::ASK_WHEN_CONNECTED;
```

Comment:
This is probably not what the user will expect. They are not connected to a server, choose to minimize instead of close and tell Mumble to remember that decision.
With the current code, Mumble will ask again, when connected to a server. That's fine. But if it is not connected to a server, Mumble will quit without asking (instead of minimizing as the user would expect).

Given the choices that we have, I think this would be our best option:
```suggestion
				Global::get().s.quitBehavior = QuitBehavior::ALWAYS_MINIMIZE;
```